doc/getting-started/apl-nuc.rst

```diff
@@ -364,6 +364,46 @@ For hugeTLB to work, you'll need to reserve huge pages:
      by using a 2M page, but make sure your huge page reservation size is
      large enough for your usage.
 
+USB Device Sharing
+==========================================
+
+The ACRN hypervisor is able to support USB device sharing
+supposing you have 2 sets of USB keyboard and mouse, 1 set is for SOS, another for UOS
+
+For SOS, boot up SOS and plug 2 sets USB keyboard and USB mouse into USB ports of device:
+
+  - run "dmesg" to check the kernel log
+
+  - record keyboard and mouse device info: 
+    
+    for example: 
+                 keyboard 1#: USB/1-3/1-3
```

Comment:
When I run the dmesg command after connecting a keyboard I don't see a message saying "keyboard 1#".   There are messages about, a new keyboard being detected, but nothing that looks like the example lines you're showing here.

Reply:
If you plug in two keyboards at the same time, how do you know which keyboard is #1 and which is #2 in your example?  Don't you have to plug them in one at a time?  

Does the ``lsusb`` command give you the information you need (instead of the rather verbose ``dmesg`` command?  Does ``1-4`` mean "bus 1, device 4" in the line ``-s 9, xhci, 1-1:1-2`` below?  (We don't seem to have any documentation about the ``acrn-dm`` parameters :(

Reply:
The best way I know of to know which port a device is connected to is to use `lsusb -t`, that's what I'd recommend we use in this guide.

Reply:
we need to unplug and plug back one by one to know which port is identify with specific number, so that people identify which will be used in UOS and modify script

Reply:
even to use isusb, we still need to run one by one to identify them,  otherwise, we need to try  which is shared to UOS by moving mouse or press keyboard, but there is an know issue now, we need to unplug and plug in keyboard and mouse for UOS after launched UOS

Reply:
The instructions need to be updated to run ``lsusb`` before plugging in any devices, then plug in one USB device at a time and use ``lsusb`` to see what was just added, noting the bus and device number.  Then use that bus and device number to specify the configuration on the acrn-dm command.

Reply:
okay, lets try to use isusb ,but i need to verify it,  i was using "dmsg"